cypress-tests/cypress/e2e/useSession.cy.ts

```diff
@@ -398,9 +398,11 @@ describe("Basic public project functionality", () => {
       .clear()
       .type("/secrets");
     cy.getDataCy("session-secrets-toggle").contains(secretName);
+    cy.waitForImageToBuild();
     cy.get(".renku-container button.btn-secondary", { timeout: TIMEOUTS.long })
       .contains("Start Session")
-      .should("exist")
+      .should("be.visible")
+      .should("be.enabled")
```

Comment:
This test is not useful if you `.click()` on a button. Cypress fails if you try to click on a `disabled` element (unless you use the `{ force: true }` option).

```suggestion
```

Reply:
Doesn't Cypress wait until the button is enabled when I add `.should("be.enabled")`?

Reply:
Cypress already waits for the button to be enabled when using `.click()`. That line does not add anything new.